app/javascript/packs/pages/provider_signup/SignupStepper.jsx

```diff
@@ -107,13 +109,13 @@ export default function SignupStepper({ steps }) {
           const allErrors = [...(errors || []), ...systemErrors].map(e =>
             e && e.message ? e.message : e
           );
+          console.log(data);
```

Comment:
Moved these up a few lines, but do we still need them?

Reply:
I'd guess we probably don't